Working with Guides guides (#2952)
dbt-labs/docs.getdbt.com: website/docs/guides/working-with-guides/cloud/1-setting-up-examples-on-cloud.md

---
title: "Setting up example projects in dbt Cloud"
id: 1-setting-up-examples-on-cloud
slug: working-with-guides/cloud/1-setting-up
description: Learn how to set up Guides example projects in dbt Cloud.
displayText: dbt Cloud setup
hoverSnippet: Learn how to set up Guides example projects in dbt Cloud.
---

> ## Getting started

Reviewer: sugg 1: removing the h2 header (otherwise when you view the page, it'll have a title followed by a header) but adding an intro like:

In this guide, you'll learn how to get set up with the new example projects in dbt Cloud.

sugg 2: i think it may be useful to include prerequisites using an h2 to ensure users know what they need to use this guide?

```suggestion
## Getting started
```

> 1. 🔀 **Fork the repo**. Go to the repo specificed as the example project in the guide you're working through, or start from [the base template](https://github.com/dbt-labs/jaffle-shop-template). Click `Use this template` and `Create new repository`.

Reviewer: bolding ui elementes and fix typo

```suggestion
 1. 🔀 **Fork the repo**: Go to the repo specified as the example project in the guide you're working through, or start from [the base template](https://github.com/dbt-labs/jaffle-shop-template). Click **Use this template** and **Create new repository**.
```

> 2. 🆕 **Create a new project** in dbt Cloud.

Reviewer: for consistency 

```suggestion
2. 🆕 **Create a new project** in dbt Cloud:
```

>    1. 🏭 **Select your warehouse**. If you don't have a warehouse, we highly suggest either working through the [GitHub Codespaces path](/guides/working-with-guides/core/1-setting-up-examples-on-core-recommended) **OR** first working through the [BigQuery Quickstart](docs/get-started/getting-started/getting-set-up/setting-up-bigquery), which can help you set up a free account on Google's BigQuery warehouse platform. If you choose the latter, use the same credentials you connect to from that Quickstart to connect your new project.

Reviewer: ```suggestion
   1. 🏭 **Select your warehouse**. If you don't have one, either go through the [GitHub Codespaces path](/guides/working-with-guides/core/1-setting-up-examples-on-core-recommended) **OR** try the [BigQuery Quickstart](docs/get-started/getting-started/getting-set-up/setting-up-bigquery), which can help you set up a free account on Google's BigQuery warehouse platform. If you opt for the latter, use the same credentials to connect your new project as you did for the Quickstart.

```

>    2. 🍴 **Connect to the forked repo** you made from the template project in Step 1. Make sure you're connecting to your fork, not the original, so that you can push changes and develop.

Reviewer: ```suggestion
   2. 🍴 **Connect to the forked repo** you made from the template project in Step 1 to push changes and develop. Make sure you connect to your fork and not the original.
```
